lib/metrics_service.rb

```diff
@@ -0,0 +1,32 @@
+require 'prometheus/client'
+
+class MetricsService
+  class << self
+    def custom_counter(name, description, labels: {})
+      counter = Prometheus::Client::Counter.new(
+        name, 
```

Comment:
I think followed the pattern ( recommended on one of the sites) to include the name of the service (i.e. pender) as a label at this label so that it is easy to separate out from other similar metrics

Reply:
That is a good idea! Will update it. Thanks Skye!

Reply:
for now I added the label `service` to the custom metric we are creating, but we can revisit it and add to the wrapper class itself if we think it makes more sense.